### PR TITLE
link sh -> bash

### PR DESCRIPTION
**Pull request by sherlockblaze**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,4 @@
+SHELL:=/bin/bash
```
Comment (ludanfeng) [inline Makefile]: Can you describe what problem has been solved?

Comment (sherlockblaze) [reply Makefile]: some environments link sh -> dash

Comment (sherlockblaze) [reply Makefile]: some environments link sh -> dash

**Comment by lileding** [comment]
The LSB standard requires a bash while only sh in POSIX or some lightweight distributors like Alpine Linux. 
And the default shell in BSD world is csh and macOS will eventually switch to zsh. 
My advise is never depend on the existence of bash, use sh always.

**Comment by sherlockblaze** [comment]
> The LSB standard requires a bash while only sh in POSIX or some lightweight distributors like Alpine Linux.
> And the default shell in BSD world is csh and macOS will eventually switch to zsh.
> My advise is never depend on the existence of bash, use sh always.

That's nice, macOS can already use zsh, but it's baetyl, i thought it's inevitable there will be compiled on 'other devices'

**Pull request closed without merging**